README.md

```diff
@@ -1,40 +1,106 @@
+* The `createBackup` step, which is **not** intended to backup user data, currently can slow down the larger the `datadirectory` gets (nextcloud/updater#507) 
+* The `deleteOldFiles` step, which does not actually touch user data, currently can slow down the larger the `datadirectory` gets (nextcloud/updater#397)
+* Multiple `*.config.php` files are currently not supported / ignored (nextcloud/updater#384)
+* In some environments, the current iterator implementation may fail (nextcloud/updater#519)
+
+## Possible future work
+
+Note these are just ideas.
+
+* Improved notifications of possible breaking changes before letting operator install major version updates
+* Additional status reporting
+* Additional error checking / failure handling
+* Additional logging
+* Additoinal performance optimizations
+* Improved reliability of the Web mode of the Updater
+* Fewer or more stringently recommended "default" update/upgrade work flow
+* Locking to specific stable release trains/majors - e.g. latest stable (N), previous stable (N-1), last supported stable (N-2)
+* Improved documentation
+* Ease of restarting failing update runs - e.g. "Update appears to be in process - leave running or reset?"
+* Rollback of incomplete Updater runs
+* Rollback of incomplete database upgrades or app updates
+* (insert your idea here)
+
 ## Background
 
 * [*How to upgrade*](https://docs.nextcloud.com/server/latest/admin_manual/maintenance/upgrade.html) in the [*Maintenance*](https://docs.nextcloud.com/server/latest/admin_manual/maintenance/index.html) chapter of the [Nextcloud Administration Manual](https://docs.nextcloud.com/server/latest/admin_manual/index.html)
 * [*What does the updater do?*](https://docs.nextcloud.com/server/latest/admin_manual/maintenance/update.html#what-does-the-updater-do)
 
 > [!NOTE]
-> The built-in Updater is primarily applicable to manual/Archive (aka: "tarball") installations. Most other installation methods (such as Docker images and Snaps) utilize their own officially supported processes for keeping Nextcloud Server up-to-date.
+> The built-in Updater is primarily applicable to manual/Archive (aka: "tarball / zip") installations. Most other installation methods (such as Docker images and Snaps) utilize their own officially supported processes for keeping Nextcloud Server up-to-date.
 > **Please follow their respective documented approaches rather than trying to run the Updater yourself.**
 
+## Installation
+
+### Requirements
+
+* An Archive based installation (i.e. **not** Docker, Snap, AIO, etc. unless you're the packager of said method and utilize the Updater internally or permit your users to utilize it).
+* The app is bundled with the installation Archive for Nextcloud Server. No additional steps are necessary to install.
+
+Note:
+
+* The Updater "app" is not a standard Nextcloud Server app so it is **not** managed via the app store.
+* The Web UI mode of Updater will not be active unless the `updatenotification` app is enabled.
```

Comment:
Out of curiosity, is there a reason for this? Is the web UI actually part of the other app?

Reply:
Yes, this is answered in the rest of the README.